rtkit-daemon.c

```diff
@@ -857,7 +865,7 @@ static void reset_known(void) {
                                 if (verify_process_user(u, p) >= 0 &&
                                     verify_process_starttime(p) >= 0 &&
                                     verify_thread_starttime(p, t) >= 0)
-                                        if (thread_reset(t->pid) >= 0) {
+                                        if (thread_reset(t->pid) >= 0 && !log_quiet) {
```

Comment:
The additional `log_quiet` check here may cause `n_demoted++` to be erroneously skipped, affecting a log message which is still issued below:

```c
syslog(LOG_NOTICE, "Demoted %u threads.\n", n_demoted)
```